website/pages/en/community.js

```diff
@@ -176,6 +176,42 @@ function Community(props) {
     </CommunityResource>
   );
 
+  const SubstrateCollaborativeLearning = () => (
+    <CommunityResource title={<translate>Substrate Collaborative Learning</translate>}>
+      <p>
+        <translate>
+          Substrate Collaborative Learning is an open video call where we learn Substrate together. We Meet every other Monday at 16:00UTC. The next several meetings are September 9th, September 23rd, October 7th, and October 21st. Ask for help, show off your project, learn Substrate, and make friends!
```

Comment:
```suggestion
          Substrate Collaborative Learning is an open video call where we learn about Substrate together. We meet every other Monday at 16:00UTC. Ask for help, show off your project, learn about Substrate, and make friends!
```

Seems like the calendar you added is enough, I don't expect you to keep updating the dates